Libraries/LibJS/Runtime/Value.cpp

```diff
@@ -494,6 +500,18 @@ i32 Value::to_i32(GlobalObject& global_object) const
     return number.as_i32();
 }
 
+u32 Value::to_u32(GlobalObject& global_object) const
+{
```

Comment:
```suggestion
u32 Value::to_u32(GlobalObject& global_object) const
{
    // 7.1.7 ToUint32, https://tc39.es/ecma262/#sec-touint32
```

(I just like putting links to the spec in these functions, makes it easier to look them up, where they're going to be used etc.)